### DISCUSSION TITLE
cy.document().invoke("write", html) throws error

### DISCUSSION THREAD

**Discussion by johannespn** [cypress-io/cypress#19270] [Questions and Help]
Hi @bahmutov,

Cypress version: 9.1.1

I'm using your https://github.com/bahmutov/cypress-ethereal-email-example/blob/main/cypress/integration/check-email.js#L45 but have the following issue:

```cy.document().invoke("write", html)``` results in 

```
Cannot set property message of [object DOMException] which has only a getter
```

```"chromeWebSecurity": false``` is set. 

Would be awesome if someone could help me out on this! :raised_hands:

**Comment by ecemac** [comment]
@johannespn Having the same issue, did you figure out the solution?
